drivers/usbdev/composite.c

```diff
@@ -38,6 +38,10 @@
 #include <nuttx/usb/usbdev.h>
 #include <nuttx/usb/usbdev_trace.h>
 
+#ifdef CONFIG_BOARD_USBDEV_SERIALSTR
+#include <nuttx/board.h>
```

Comment:
could you point out where composite driver use the interface from nuttx/board.h?

Reply:
For calling `board_usbdev_serialstr()` at line 385

Reply:
this line need intent too:
```suggestion
#  include <nuttx/board.h>
```